client/my-sites/domains/domain-management/list/list-header.jsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { CompactCard } from '@automattic/components';
+import InfoPopover from 'components/info-popover';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class ListHeader extends React.PureComponent {
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<CompactCard className="list-header">
+				<div className="list__domain-link" />
+				<div className="list__domain-transfer-lock">
+					{ translate( 'Transfer lock' ) }
+					<InfoPopover iconSize={ 18 }>
+						{ translate(
+							'When enabled, a transfer lock prevents your domain from being transferred to another ' +
+								'provider. Sometimes the transfer lock cannot be disabled, such as when a domain ' +
+								'is recently registered.'
+						) }
+					</InfoPopover>
+				</div>
+				<div className="list__domain-privacy">
+					{ translate( 'Privacy' ) }
+					<InfoPopover iconSize={ 18 }>
+						{ translate(
+							'Enabling domain privacy protection hides your contact information from public view. ' +
+								'For some domain extensions, such as some country specific domain extensions, ' +
+								'privacy protection is not available.'
+						) }
+					</InfoPopover>
+				</div>
+				<div className="list__domain-auto-renew">
+					{ translate( 'Auto-renew' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Auto-renew (…)' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/new-domains-header)